### DISCUSSION TITLE
Using websockets and Laravel Echo

### DISCUSSION THREAD

**Discussion by ivdimova** [filamentphp/filament#10096] [General]
Hi,
I need to use events, websockets and Echo to track when my project get a response from external API. However I can't find information how to use Echo with Filament, as the usual bootstrap.js file is not included. I've uncommented the Echo config in `config\filament.php`, but can't figure out where and how to listen to an event? Thanks in advance!

**Comment by hasanbasri1993** [comment]
here maybe help
https://laraveldaily.com/post/configure-laravel-reverb-filament-broadcasting
